draft-jasdips-regext-rdap-rpki.md

```diff
@@ -686,6 +696,230 @@ Here is an elided example for an autonomous system number object with ASPAs:
+    * publicKey -- a string representation of the public key
+* autnum -- an unsigned 32-bit integer representing the autonomous system number that the router emits secure route
+  advertisements on behalf of ([@!RFC8209, section 3.1.3.5])
+* notValidBefore -- a string that contains the time and date in Zulu (Z) format with UTC offset of 00:00 ([@!RFC3339]),
+  representing the not-valid-before date of the certificate
+* notValidAfter -- a string that contains the time and date in Zulu (Z) format with UTC offset of 00:00 ([@!RFC3339]),
+  representing the not-valid-after date of the certificate
+* events -- see [@!RFC9083, section 4.5]
+* links -- links ([@!RFC9083, section 4.2]) for "self", and "related" to autonomous system number and IRR (when defined)
+  objects
+* remarks -- see [@!RFC9083, section 4.3]
+
+Here is an elided example of a BGPSec Router Certificate object in RDAP:
+
+```
+{
+  "objectClassName": "rpki bgpsec router cert",
+  "handle": "ABCD",
+  "serialNumber": "1234",
+  "issuer": "CN=ISP-CA",
+  "signatureAlgorithm": "ecdsa-with-SHA256",
+  "subject": "CN=ROUTER-ASN-65536",
+  "subjectPublicKeyInfo":
+  {
+    "publicKeyAlgorithm": "id-ecPublicKey",
+    "publicKey": "...",
+  }
+  "autnum": 65536,
+  "notValidBefore": "2024-04-27T23:59:59Z",
+  "notValidAfter": "2025-04-27T23:59:59Z"
+  "events":
+  [
+    {
+      "eventAction": "registration",
+      "eventDate": "2024-01-01T23:59:59Z"
+    },
+    ...
+  ],
+  "links":
+  [
+    {
+      "value": "https://example.net/rdap/rpkiBgpsecRouterCert/65536",
+      "rel": "self",
+      "href": "https://example.net/rdap/rpkiBgpsecRouterCert/65536",
+      "type": "application/rdap+json"
+    },
+    {
+      "value": "https://example.net/rdap/rpkiBgpsecRouterCert/65536",
+      "rel": "related",
+      "href": "https://example.net/rdap/autnum/65536",
+      "type": "application/rdap+json"
+    },
+    ...
+  ],
+  "remarks":
+  [
+    {
+      "description": [ "A BGPSec Router Certificate object in RDAP" ]
+    }
+  ]
+}
+```
+
+## Lookup
+
+The resource type path segment for exact match lookup of a BGPSec Router Certificate object is "rpkiBgpsecRouterCert".
+
+The following lookup path segment is defined for a BGPSec Router Certificate object:
+
+Syntax: rpkiBgpsecRouterCert/<autonomous system number>
+
+For example:
+
+```
+https://example.net/rdap/rpkiBgpsecRouterCert/65536
+```
+
+## Search
+
+The resource type path segment for searching BGPSec Router Certificate objects is "rpkiBgpsecRouterCerts".
+
+The following search path segments are defined for BGPSec Router Certificate objects:
+
+Syntax: rpkiBgpsecRouterCerts?handle=<handle search pattern>
+
+Syntax: rpkiBgpsecRouterCerts?issuer=<certificate issuer search pattern>
+
+Syntax: rpkiBgpsecRouterCerts?subject=<certificate subject search pattern>
+
+Searches for BGPSec router certificate information by handle are specified using this form:
+
+rpkiBgpsecRouterCerts?handle=XXXX
```

Comment:
For search patterns, are you considering the partial string searching as specified in Section 4.1 of RFC 9082?

Reply:
> For search patterns, are you considering the partial string searching as specified in Section 4.1 of RFC 9082?

We should. Good point.